common/changes/@itwin/core-frontend/ao-nonpickable-fixes_2022-01-24-14-59.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
```

Comment:
Run rushchange to generate changelog for @bentley package then delete this @itwin changelog.